docs/Installation-Guide.rst

```diff
@@ -769,6 +768,103 @@ gcc
 
 Also, you may want to read `gcc Tips <./gcc-Tips.rst>`__.
 
+Build C++ Unit Tests
+~~~~~~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^
+
+On Windows a C++ unit tests of LightGBM can be built using **CMake** and **VS Build Tools**.
```

Comment:
```suggestion
On Windows, C++ unit tests of LightGBM can be built using **CMake** and **VS Build Tools**.
```

Reply:
@jameslamb Don't you mind if I improve this wording in a separate PR for all such occurrences? Just like https://github.com/microsoft/LightGBM/pull/3887#issue-564725297.

![image](https://user-images.githubusercontent.com/25141164/114306317-46718d80-9ae4-11eb-976d-b377f058c97a.png)

![image](https://user-images.githubusercontent.com/25141164/114306321-4c676e80-9ae4-11eb-9156-b93d567f1fc0.png)

Reply:
Those two examples you circled are correct and don't need to be changed, but "a C++ unit tests" is not correct. "a" refers to one thing but "tests" is plural. If you want this language to be similar to those examples, it might be "On Windows, a version of LightGBM instrumented for testing"

Reply:
Yeah, I meant comma after OS. I think I'd better push your suggestion right now.